composer.json

```
@@ -8,7 +8,7 @@
         }
     },
     "require"    : {
-        "php": ">=5.4"
+        "php": "5.3.10"
```

```suggestion
        "php": "^5.3.10|^7.0"
```

Never use an unbounded range for PHP versions (`>=5.4` is unbounded and claims that all PHP versions newer than 5.4 are supported, nothing is PHP 8 ready right now).  And without a version comparison symbol, you're saying that only 5.3.10 is allowed.